### PR TITLE
#1 Print then with substitutions

### PR DESCRIPTION
**Pull request by ihaiker**

### CODE REVIEW

File: src/main.rs [hunk 1]
```diff
@@ -310,7 +310,7 @@ fn process_events(
                         println!(
                             "[{}] Execute: {:?} from {:?}",
                             Utc::now().format("%Y-%m-%d %H:%M:%SZ"),
-                            ifft.then,
+                            ifft.then.replace("{{}}",path.as_ref().unwrap().to_str().unwrap()),
```
Comment (braincore) [inline src/main.rs]: Could you use `expect` as is done on line 596? Or better yet, refactor substitution into a separate function.

**Comment by braincore** [comment]
Good idea! I have one comment, and then could you please improve the commit message to something like "Print `then` with substitutions"

File: src/main.rs (hunk 1 above)
Comment (ihaiker) [reply src/main.rs]: Good idea, I've added a new method.

**Comment by ihaiker** [comment]
OK

**Pull request closed without merging**